src/core/address.js

```diff
@@ -6,6 +6,7 @@ export type Address = {|
   +repositoryName: string,
   +pluginName: string,
   +id: string,
+  +type: string,
 |};
 
 export interface Addressable {
```

Comment:
nit: on line 106, the comment can be changed from "three" to "four". Likewise, on line 27, maybe "nested map".

Reply:
Done